.github/workflows/docker-build-test-upload.yml

```diff
@@ -11,10 +11,19 @@ on:
         description: Parent image name
         required: true
         type: string
+      parent-variant:
+        description: Parent variant tag prefix
+        required: false
+        type: string
+        default: default
       image:
         description: Image name
         required: true
         type: string
+      variant:
+        description: Variant tag prefix
+        required: true
+        type: string
```

Comment:
I suggest making default the default value here as well.

Reply:
Added the default.